pages/api/upload_album.py

```
@@ -942,27 +937,10 @@ def delete_all_trash():
-
-        return send_from_directory(
-            str(LOCAL_STORAGE_PATH),
-            filename,
-            mimetype='image/jpeg'
-        )
-    except Exception as e:
-        print(f"[ERROR] serve_dataclient failed: {str(e)}")
-        return jsonify({'error': str(e)}), 500
+def serve_file(filename):
+    print(f"Requesting file: {filename}")
+    print(f"Full path: {os.path.join(Config.BASE_STORAGE_PATH, filename)}")
```

Ruff (0.7.0)

`Config` may be undefined, or defined from star imports

(F405)